resources/todomvc/architecture-examples/react/src/todo/components/input.jsx

```diff
@@ -26,7 +26,7 @@ export function Input({ onSubmit, placeholder, label, defaultValue, onBlur }) {
 
     const handleKeyDown = useCallback(
         (e) => {
-            if (e.keyCode === 13) {
+            if (e.key.match(/Enter/i)) {
```

Comment:
I noticed that the value for `key` we get from the runner is `ENTER`, but it should really be `Enter` ([see MDN](https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values)). IMO the runner should be fixed at https://github.com/WebKit/Speedometer/blob/6523f664973df6bffa2dd24836f8b479f1d0f241/resources/benchmark-runner.mjs#L110

Reply:
merged main with the fix